marimbabot_planning/src/actions/hit_sequence.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
```

Comment:
What is this `sequence` supposed to represent? It seems an insufficient representation either way if you want to support playing multiple notes with different durations in a score.

Assuming each int is basically a midi tone, you would only have to add an onset time per note to get a generic "midi-style" representation.

Additionally, you need to add a comprehensible description because this is the only text that defines what exactly the fields mean. (The implementation has to implement what the definition says, not the other way around :))

Reply:
You are right, I changed it to a list of strings for now.